src/design-system-team.md

```diff
@@ -29,6 +29,7 @@ If you want to contact the team you can [get in touch via email or Slack](/get-i
 - Kam Nijjar – Associate Delivery Manager
 - Katrina Birch – User Researcher
 - Kim 'beeps' Grey – Frontend Developer
+- Mia Allers – Graphic Designer  
```

Comment:
I'm not sure how much this matters but I can't remember if Mia is a senior or lead graphic designer. It's not exactly a show stopper but other folks have senior in their title on this page.

Reply:
Good spot! She is senior...will amend 👍🏻